techhive/seeders/internships.json

```diff
@@ -18,11 +18,69 @@
     },
     {
       "title": "Product Design Intern Apple",
-      "picture": "https://example.com/images/internship3.jpg",
-      "link": "https://example.com/internship3",
+      "picture": "https://jobs.apple.com/",
+      "link": "/Apple-Logo.jpg",
       "description": "Work on data analysis and machine learning projects under the guidance of experienced data scientists.",
-      "company": "Apply",
+      "company": "Apple",
       "category": "Junior friendly, Senior friendly"
-    }
+    },
+    {
+      "title": "Google Engineering Practicum Internship",
+      "picture": "/Google-Logo.jpg",
+      "link": "https://careers.google.com/students/",
+      "description": "The Google Engineering Practicum Internship is a summer program designed for first and second-year undergraduate students with a passion for computer science. As an Engineering Practicum intern, you will have the opportunity to work on real engineering projects, collaborate with experienced engineers, and gain valuable hands-on experience in software development. Join Google and be part of a team that is shaping the future of technology.",
+      "company": "Google",
+      "category": "Freshman Friendly, Sophomore Friendly"
+      },
+      {
+        "title": "Meta University Engineering Internship",
+        "picture": "/Facebook-Logo.jpg",
+        "link": "https://www.facebook.com/careers/students-and-grads/students",
+        "description": "The Meta University Engineering Internship is a paid summer internship program that offers hands-on experience in software engineering at Facebook. Open to undergraduate students, this internship provides an opportunity to work on impactful projects, collaborate with talented engineers, and learn from industry leaders. Join Meta University and gain the skills and knowledge to make a difference in the world of technology.",
+        "company": "Meta",
+        "category": "Freshman Friendly, Sophomore Friendly"
+      },
+      {
+        "title": "Amazon Software Development Engineer Internship",
+        "picture": "/Amazon-Logo.jpg",
+        "link": "https://www.amazon.jobs/en/teams/internships-for-students",
+        "description": "The Amazon Software Development Engineer Internship offers a challenging and rewarding experience for undergraduate students interested in software development. As an intern, you will work on projects that have a direct impact on Amazon's customers, collaborate with talented engineers, and learn from experienced mentors. Join Amazon and be part of a team that is revolutionizing e-commerce and technology.",
+        "company": "Amazon",
+        "category": "Sophomore Friendly, Junior Friendly"
```

Comment:
just an idea for future, maybe we can make this an array so it's easier to do multi-select filter in the future `"category": ["Sophomore Friendly", "Junior Friendly"]`

Reply:
Okay sure, that makes sense